rule-engine/build.gradle

```diff
@@ -0,0 +1,27 @@
+plugins {
+    id 'java'
+}
+
+group 'org.opensearch.ruleengine'
```

Comment:
Is there a clearer group you can use here? We actually have a different PR ( #4446) with rule evaluation?

```
group 'org.opensearch.securityanalytics.rulengine
```

???